pkg/console/subresource/route/http2_test.go

```
@@ -0,0 +1,358 @@
+	return &corev1.Secret{
+		Data: map[string][]byte{
+			"tls.crt": []byte(cert.Certificate),
+			"tls.key": []byte(cert.Key),
+		},
+	}
+}
+
+func newFakeSecretLister(secrets ...*corev1.Secret) corev1listers.SecretLister {
+	indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
+	for _, s := range secrets {
+		indexer.Add(s.DeepCopy())
```

golangci-lint (2.12.2)

[error] Error return value of `indexer.Add` is not checked

(errcheck)